presto-parser/src/main/java/com/facebook/presto/sql/tree/Call.java

```diff
@@ -45,7 +45,7 @@ public Call(Optional<NodeLocation> location, QualifiedName name, List<CallArgume
         this.arguments = ImmutableList.copyOf(requireNonNull(arguments, "arguments is null"));
     }
 
-    public QualifiedName getName()
+    public QualifiedName getQualifiedName()
```

Comment:
Could you at least mention in the commit message that you renamed getName() to getQualifiedName()? Or extract the commit? It seems like what you're returning is changing but it actually isn't.